### ISSUE TITLE
Understanding Self-Supervised Learning Dynamics without Contrastive Pairs

### ISSUE THREAD

**Issue by SerialLain3170** [SerialLain3170/nagato#85]
## Overview

- Authors theoretically shows the influence of multiple factors (Exponential moving average, learning rate for predictor and weight decay) in the setting of self-supervised learning without negative pairs
  - Larger learning rate for predictor leads to a smaller basin of attraction of the collapsed branch by flattening the parabola
  - The final value of the online network does not grow even if the value of predictor does
  - Weight decay allows final value of the online network to also model invariance to augmentations correctly
  - Larger weight decay expands the collapsed regions
  - Strong weight decay leads to unavoidable collapse

![Screen Shot 2021-11-14 at 16 25 02](https://user-images.githubusercontent.com/32360147/141671824-ccd28880-c6c6-40f7-bba1-64429b657abd.png)

- Authors also propose DirectPred, which simply estimates the covariance matrix of inputs of predictor and sets the weights of predictor

**Comment by SerialLain3170** [comment]
- DirectPred is competitive to previous methods that require gradient to the weights of predictor

![Screen Shot 2021-11-14 at 16 26 16](https://user-images.githubusercontent.com/32360147/141671843-9d8abd8a-3f65-4ec9-a71d-9e66b64b6436.png)

**Comment by SerialLain3170** [comment]
## Supplemental Information

[Paper Link](http://proceedings.mlr.press/v139/tian21a/tian21a.pdf)

### Authors
Yuandong Tian, Xinlei Chen, Surya Ganguli

### Organization
Facebook AI Research
Stanford University

### Conference
ICML 2021
